### PR TITLE
Module scope compatibility

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Explain Module scope compatibility

### CODE REVIEW

**Comment by matks** [comment]
@SimonGrn this should stop the confusion about modules being tested against unreleased prestashop versions

**Comment by SimonGrn** [comment]
Thanks, that gives us a link to give to contributors.

File: src/content/1.7/contribute/contribution-guidelines/project-modules.md [hunk 1]
```diff
@@ -43,6 +43,35 @@ They can easily be applied by using the `header-stamp` binary from [PrestaShop m
...
+
+PrestaShop and its modules follows [SemVer](https://semver.org/). This means that contributions should strive not to introduce breaking compatibility changes. Therefore, each Pull Request must not introduce such changes, unless aiming to release a new major version.
+
+It is possible to introduce a change that is not backward compatible into the code for a minor or patch version, but it must be for a good reason.
+
+Valid reasons include, but not only:
+- fixing a security issue
+- fixing a major issue that cannot be fixed in a backward compatible manner
+- introducing a major new feature of high value
+
```
Comment (Progi1984) [inline src/content/1.7/contribute/contribution-guidelines/project-modules.md:58]: @PrestaShop/prestashop-core-developers Question : Changing the property `ps_versions_compliancy` is a BC or not ?

Comment (PierreRambaud) [reply src/content/1.7/contribute/contribution-guidelines/project-modules.md:58]: Should not be a BC breaks as the Core won't install an incompatible module :thinking:

**Comment by matks** [comment]
@PierreRambaud @Progi1984 I need your approval 😁

**Pull request merged**